curvefs/src/client/fuse_client.cpp

```diff
@@ -1149,8 +1149,38 @@ CURVEFS_ERROR FuseClient::FuseOpSetXattr(fuse_req_t req, fuse_ino_t ino,
     return CURVEFS_ERROR::OK;
 }
 
+CURVEFS_ERROR FuseClient::FuseOpRemoveXattr(fuse_req_t req, fuse_ino_t ino,
+    const char *name) {
+    std::string strname(name);
+    VLOG(1) << "FuseOpRemoveXattr ino: " << ino << ", name: " << name;
+
+    if (option_.fileSystemOption.disableXAttr || IsSpecialXAttr(name)) {
```

Comment:
We should handle the curve specified xattr even if the `disableXAttr` is true, so maybe the judge is wrong.

Reply:
fixed